src/framework/theme/components/menu/menu-item.component.html

```diff
@@ -9,7 +9,8 @@
    [attr.target]="menuItem.target"
    [attr.title]="menuItem.title"
    [class.active]="menuItem.selected"
-   (mouseenter)="onHoverItem(menuItem)">
+   (mouseenter)="onHoverItem(menuItem)"
+   (click)="$event.preventDefault(); onItemClick(menuItem);">
```

Comment:
I guess it should not emit the click for a reason - we navigate to a page so most likely the click won't make any difference as the handler will be destroyed.